mon-pix/app/routes/campaigns/assessment/results.js

```diff
@@ -42,4 +43,16 @@ export default class ResultsRoute extends Route {
       } else throw error;
     }
   }
+
+  async afterModel({ campaignParticipationResult }) {
+    if (!this.featureToggles.featureToggles?.isAutoShareEnabled) {
```

Comment:
A-t-on vraiment besoin du point d'interrogation
```suggestion
    if (!this.featureToggles.featureToggles.isAutoShareEnabled) {
```
On renverra undefined donc non si on ne trouve rien, donc pas d'erreur.

Reply:
Si tu cherches dans le code ils ont tous l'optional chaining